editor/src/clj/editor/animation_set.clj

```diff
@@ -191,4 +191,4 @@
     :sanitize-fn sanitize-animation-set
     :node-type AnimationSetNode
     :ddf-type Rig$AnimationSetDesc
-    :view-types [:cljfx-form-view]))
+    :view-types [:cljfx-form-view :text]))
```

Comment:
hmm, it isn't protobuf?

Reply:
Yeah, it is Protobuf-based, but we allow you to inspect its text representation. We do this for all the other Protobuf-based types. For some reason it was missing for `.animationset`.